tests/acceptance/account_details-test.js

```diff
@@ -0,0 +1,86 @@
+import Ember from 'ember';
+import { module, test } from 'qunit';
+import startApp from '../helpers/start-app';
+import {make} from 'ember-data-factory-guy';
+import { mockFindAll } from 'ember-data-factory-guy';
+
+var App, user, organisation, organisationsUser, gogo_van, order, pkg, ordersPackage, gcOrganisations;
+
+module('Acceptance | Account Details Page', {
+  beforeEach: function() {
+    window.localStorage.authToken = '"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjo2LCJpYXQiOjE1MTg3NzI4MjcsImlzcyI6Ikdvb2RDaXR5SEsiLCJleHAiOjE1MTk5ODI0Mjd9.WdsVvss9khm81WNScV5r6DiIwo8CQfHM1c4ON2IACes"';
+    App = startApp();
+    user = make("user");
+    organisation = make("organisation");
+    organisationsUser = make("organisations_user", {user_id: user.id, organisation_id: organisation.id});
+    pkg = make('package');
+    ordersPackage = make("orders_package", { quantity: 1, state: "requested", package: pkg, packageId: pkg.id, order: order});
+    order = make("order", { state: "draft", created_by_id: user.id });
+    gogo_van = make("gogovan_transport");
+    gcOrganisations = make("gc_organisation", {nameEn: "GCC club"});
+
+    $.mockjax({url: "/api/v1/available_*", type: 'GET', status: 200, responseText:["2018-06-14", "2018-06-15", "2018-06-16", "2018-06-19", "2018-06-20", "2018-06-21"]});
```

Comment:
Mockjax mocks should be cleared after the tests are run, otherwise they add up and might end up conflicting with other tests later. Plus this adds the same mock to the list every time the test is run, which isn't needed.

You can use `$.mockjax.clear()` to clear everything (might affect other tests)
or you can clear mocks individually
`
const mockId = $.mockjax("/api/endpoint", ...);
$.mockjax.clear(mockId)
`